.github/workflows/ci.yml

```diff
@@ -55,12 +63,28 @@ jobs:
       - name: Check auth
         run: tb --host ${{ secrets.tb_host }} --token ${{ secrets.tb_admin_token }} auth info
 
+      - name: Try delete previous Environment
```

Comment:
Let's see how workflows evolve but it would be nice to create a proper github action to use our cli so we can encapsulate this kind of things as they getting complex (also getting advantage of our cli docker image).
Also taking into account how to reuse for other providers